lib/std/child_process.zig

```diff
@@ -32,6 +32,8 @@ pub const ChildProcess = struct {
     stdout: ?File,
     stderr: ?File,
 
+    overlapped: if (builtin.os.tag == .windows) [2]windows.OVERLAPPED else struct { } = undefined,
```

Comment:
I've made these `OVERLAPPED` instances a member of `ChildProcess` because they need to outlive the lifetime of an asynchronous read operation, but, if an error occurs in `collectOutputWindows`, then a pending async operation could still be in progress but the `OVERLAPPED` instance on the stack will be freed upon return of the function.  We can either implement a cancel mechanism, or, put the `overlapped` data in a place that outlives the `stdout`/`stderr` pipes.